src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketBuildTrigger.java

```diff
@@ -214,7 +216,16 @@ protected Job asJob() {
         return retrieveScheduleJob(this.job).scheduleBuild2(0,
                 new CauseAction(cause),
                 new ParametersAction(new ArrayList(values.values())),
-                new RevisionParameterAction(cause.getSourceCommitHash()));
+                new RevisionParameterAction(cause.getSourceCommitHash(), getBitbucketRepoUrl(cause.getRepositoryOwner(), cause.getRepositoryName())));
```

Comment:
Wait a second - Are you saying that if we specify our url here, that the function won't break other git calls? Meaning if we set the url to what we're using, then future calls to git or checkout will work?

Reply:
Yeah, if we set the url here, git scm ignores the commit that is specified unless the repo its checking out has the same url as the one specified.